### PR TITLE
fix(ui5-upload-collection): add tooltip to Edit button

### PR DESCRIPTION
**Pull request by gmkv**

### CODE REVIEW

File: packages/fiori/src/UploadCollection.js [hunk 1]
```diff
@@ -141,7 +141,7 @@ const metadata = {
 		 * Fired when the Delete button of any item is pressed.
 		 * <br><br>
 		 * <b>Note:</b> A Delete button is displayed on each item,
-		 * when the <code>ui5-upload-collection</code> <code>mode</code> property is set to <code>Delete</code>.
+		 * when the <code>ui5-upload-collection</code> <code>mode</code> attribute is set to <code>Delete</code>.
```
Comment (fifoosid) [inline packages/fiori/src/UploadCollection.js:144]: mode is a property as well

File: packages/fiori/src/UploadCollection.js [hunk 2]
```diff
@@ -73,7 +73,7 @@ const metadata = {
 
 		/**
 		 * By default there will be drag and drop overlay shown over the <code>ui5-upload-collection</code> when files
-		 * are dragged. If you don't intend to use drag and drop, set this property to <code>true</code>
+		 * are dragged. If you don't intend to use drag and drop, add this attribute.
```
Comment (fifoosid) [inline packages/fiori/src/UploadCollection.js:76]: If you don't intend to use drag and drop, set this property.

**Review by fifoosid — CHANGES_REQUESTED** [review]
From technical point of view looks good to me, just some docs.

File: packages/fiori/src/UploadCollection.js (hunk 1 above)
Comment (gmkv) [reply packages/fiori/src/UploadCollection.js]: Done.

File: packages/fiori/src/UploadCollection.js (hunk 2 above)
Comment (gmkv) [reply packages/fiori/src/UploadCollection.js]: Done.

**Review by fifoosid — APPROVED** [review]

**Pull request merged**